### PR TITLE
python3Packages.py17track: init at 3.2.1

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python library to track package info from 17track.com

https://github.com/bachya/py17track

This is a Home Assistant dependency.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 128564` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>home-assistant</li>
  </ul>
</details>
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.py17track</li>
    <li>python39Packages.py17track</li>
  </ul>
</details>

**Comment by dotlambda** [comment]
I don't see an upstream PR updating py17track. Maybe we should pin it to 3.2.1?

**Comment by fabaff** [comment]
>I don't see an upstream PR updating py17track.

https://github.com/home-assistant/core/pull/51057

> Maybe we should pin it to 3.2.1?

I guess that we have to not break the tests.

File: pkgs/development/python-modules/py17track/default.nix [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+  checkInputs = [
+    aresponses
+    pytest-asyncio
+    pytestCheckHook
+  ];
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace ">=19.3,<21.0" ">=19.3,<22.0"
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/py17track/default.nix:47]:
```suggestion
      --replace "attrs = ">=19.3,<21.0"" "attrs = ">=19.3"
```

Comment (fabaff) [reply pkgs/development/python-modules/py17track/default.nix]: Changed, thanks.

**Review by SuperSandro2000 — COMMENTED** [review]
Is this an init or a downgrade?

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 128564` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.py17track</li>
    <li>python39Packages.py17track</li>
  </ul>
</details>

**Comment by fabaff** [comment]
> Is this an init or a downgrade?

It's an `init` but I downgraded to 3.2.1 instead of 3.3.0 as the upstream PR that introduced 3.3.0 is stuck.

**Comment by fabaff** [comment]
This module was already integrated.

**Pull request closed without merging**